### PR TITLE
docs: clean up news for the release

### PR DESCRIPTION
**Pull request by bms63**
Just a quick clean up of the news.  

* A few items were duplicated (bad merge??)
* Added issue numbers to entries missing issue numbers.
* Separated out Bug Fixes

@elimillera @EeethB - after this merge we can ship to CRAN.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,60 +1,46 @@
 # xportr (development version)
```
Comment (elimillera) [inline NEWS.md:1]:
```suggestion
# xportr 0.4.0
```

**Review by elimillera — COMMENTED** [review]
Can we update the header here for the version? Or do we have CI/CD to do that?

File: NEWS.md (hunk 1 above)
Comment (bms63) [reply NEWS.md:1]: I believe you can use https://usethis.r-lib.org/reference/use_version.html to update the news and description file for the release.

File: NEWS.md [hunk 2]
```diff
@@ -1,60 +1,46 @@
...
-
-* `xportr_metadata()` can set `verbose` for a whole pipeline, i.e. setting `verbose` in `xportr_metadata()` will populate to all `xportr` functions.  (#151)
+## New Features
 
-* All `xportr` functions now have `verbose = NULL` as the default (#151)
-
-* Bug fix for domain filtering (#137)
-
-## Documentation
+* All core functions can be run together by using new function `xportr()` (#137)
```
Comment (averissimo) [inline NEWS.md:6]: Can we remove the empty lines between sub-sections?

It seems odd.

Comment (bms63) [reply NEWS.md:5]: i have prettified

Comment (averissimo) [reply NEWS.md:5]: Oh, I meant the empty lines between items (starting with `*`). The ones between headings and the first item seem fine.

image

I only mentioned this as previous releases don't have it :shrug: This is very minor though, release is looking goooood!

Comment (bms63) [reply NEWS.md]: ah i see!!  updated!  I appreciate the detailed being paid attention to the changelog!!

**Review by elimillera — APPROVED** [review]

**Pull request merged**